### PR TITLE
DOC use npm ci instead on npm install

### PR DESCRIPTION
**Pull request by adrinjalali**
`package.json` and `package-lock.json` are not in sync, and doing an `npm install` wouldn't work. We either need to sync them up, or ask users to install with `npm ci` instead. This PR suggests the latter.

Note that I'm an absolute newbie with JS, and took me a while to figure this out with the help of a friend of mine, and I'm not sure which solution we should take.

### CODE REVIEW

**Review by Pierrci — COMMENTED** [review]
Thanks @adrinjalali! Given the very early phase of sveltekit (with things that tend to break very often between versions), your solution is probably the best one for now, wdyt @mishig25?

File: js/README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ TODO: write more documentation
 
 ## How to develop
 
-Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+Once you've created a project and installed dependencies with `npm ci` (or
+`pnpm install` or `yarn`), start a development server:
```
Comment (mishig25) [inline js/README.md:18]:
```suggestion
`pnpm install --frozen-lockfile` or `yarn install --frozen-lockfile`), start a development server:
```

**Comment by mishig25** [comment]
I think it is a great suggestion! locally, I have always been using `npm ci`

**Comment by adrinjalali** [comment]
Not sure why CI fails with

```
Logging into your Netlify account...
Opening https://app.netlify.com/authorize?response_type=ticket&ticket=5e9f1e4a098120f6ec040eaa802b8823
- Waiting for authorization...
 ›   Error: Timed out waiting for authorization. If you do not have a Netlify account, please create one at https://app.netlify.com/signup, then run netlify login again.
Error: Process completed with exit code 1.
```

**Review by mishig25 — APPROVED** [review]

**Comment by mishig25** [comment]
the CI is failing because, currently the CI doesn't work when you try to merge from a fork

**Pull request merged**